### PR TITLE
Migrated NeoSelect to tailwindcss

### PR DESCRIPTION
**Pull request by shashkovdanil**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #8222 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [x] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at b64b6d2</samp>

Refactored NeoSelect component style with Tailwind CSS and oruga-ui theme. Simplified SCSS code and removed unused import in `NeoSelect.scss`.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at b64b6d2</samp>

> _Sing, O Muse, of the skillful coder who refined_
> _The NeoSelect component, a marvel of design,_
> _And with the @apply directive and theme function wrought_
> _A splendid style with Tailwind CSS and oruga-ui, as he thought._

### CODE REVIEW

File: libs/ui/src/components/NeoSelect/NeoSelect.scss [hunk 1]
```diff
@@ -2,22 +2,18 @@
...
 
 $select-padding: 0.5rem;
 $select-height: 3rem;
 $select-border-radius: 0;
 
 @import '@oruga-ui/oruga-next/src/scss/components/select.scss';
+
 .neo-select.o-ctrl-sel {
-  display: inline-block;
-  max-width: 100%;
```
Comment (roiLeo) [inline libs/ui/src/components/NeoSelect/NeoSelect.scss:14]: already doable with `expanded` prop

**Review by roiLeo — APPROVED** [review]
let's try to use select [class props](https://oruga.io/components/Select.html#class-props) when we can (`o-sel` => `selectClass`)

lgtm for now

**Comment by roiLeo** [comment]
> Could you share some code example, please? I don't know Oruga well yet

see for example `NeoTooltip` or `NeoSkeleton`, you can define some wrapper class of component

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 6.5 USD/DOT ~ 3.077 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0x9cfe44f58e51cfe72b795f2a69ad9f83ddf1b4908420e98956fe1d3a90f2c480](https://polkadot.subscan.io/extrinsic/0x9cfe44f58e51cfe72b795f2a69ad9f83ddf1b4908420e98956fe1d3a90f2c480)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**